README.md

```diff
@@ -61,16 +73,16 @@ Given a set of Ingress resources, `ingress2gateway` will generate a Gateway with
 Although most annotations are ignored, this project includes experimental
 support for the following annotations:
 
-* kubernetes.io/ingress.class: Same behavior as the `ingressClassName` field above, if specified this value will be used as the `gatewayClassName` set on the corresponding generated Gateway.
+* `kubernetes.io/ingress.class`: Same behavior as the `ingressClassName` field above, if specified this value will be used as the `gatewayClassName` set on the corresponding generated Gateway.
 
 #### ingress-nginx:
 
-* nginx.ingress.kubernetes.io/canary: If set to `true` will enable weighting backends.
-* nginx.ingress.kubernetes.io/canary-by-header: If specified, the value of this annotation is the header name that will be added as a HTTPHeaderMatch for the routes generated from this Ingress. If not specified, no HTTPHeaderMatch will be generated.
-* nginx.ingress.kubernetes.io/canary-by-header-value: If specified, the value of this annotation is the header value to perform an `HeaderMatchExact` match on in the generated HTTPHeaderMatch.
-* nginx.ingress.kubernetes.io/canary-by-header-pattern: If specified, this is the  pattern to match against for the HTTPHeaderMatch, which will be of type `HeaderMatchRegularExpression`.
-* nginx.ingress.kubernetes.io/canary-weight: If specified and non-zero, this value will be applied as the weight of the backends for the routes generated from this Ingress resource.
-* nginx.ingress.kubernetes.io/canary-weight-total
+* `nginx.ingress.kubernetes.io/canary`: If set to `true` will enable weighting backends.
+* `nginx.ingress.kubernetes.io/canary-by-header`: If specified, the value of this annotation is the header name that will be added as a HTTPHeaderMatch for the routes generated from this Ingress. If not specified, no HTTPHeaderMatch will be generated.
+* `nginx.ingress.kubernetes.io/canary-by-header-value`: If specified, the value of this annotation is the header value to perform an `HeaderMatchExact` match on in the generated HTTPHeaderMatch.
+* `nginx.ingress.kubernetes.io/canary-by-header-pattern`: If specified, this is the  pattern to match against for the HTTPHeaderMatch, which will be of type `HeaderMatchRegularExpression`.
+* `nginx.ingress.kubernetes.io/canary-weight`: If specified and non-zero, this value will be applied as the weight of the backends for the routes generated from this Ingress resource.
+* `nginx.ingress.kubernetes.io/canary-weight-total`
```

Comment:
Not sure how I missed this the first time, but this one is lacking a corresponding description

Reply:
Yep, add it later.